glib/src/value.rs

```diff
@@ -246,9 +351,9 @@ impl Value {
     }
 
     /// Tries to transform the value into a value of the target type
-    pub fn transform<T: StaticType + SetValue>(&self) -> Option<Value> {
+    pub fn transform<T: ValueType>(&self) -> Option<Value> {
```

Comment:
Maybe that would be worth changing the API and returning a `Result` so that we can easily `Display` the type mismatch.

Reply:
Good idea